tenants-certified-attributes-updater/src/main/scala/it/pagopa/interop/tenantscertifiedattributesupdater/util/Utils.scala

```diff
@@ -77,7 +83,7 @@ object Utils {
     val fromTenant: Map[PersistentExternalId, List[AttributeInfo]] =
       tenants
         .map(tenant =>
-          tenant.externalId -> tenant.attributes.flatMap(AttributeInfo.addRevocationTimeStamp(attributesIndex))
+          tenant.externalId -> tenant.attributes.flatMap(AttributeInfo.addRevocationTimeStamp(admittedAttributesIndex))
```

Comment:
This flatMap is in `InterableOnce`, it's a `List.flatMap(Option)`, is it desired?